_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "Working in the Open: Part 2"
+ref: working-in-the-open-p2
+lang: en
+author: "Guillaume Charest, in collaboration with the IT Strategy team"
+date: "2021-06-08"
+excerpt_separator: <!--more-->
+---
+This series of blog posts will explore the concept of "Working in the Open".
+In this second entry, we will attempt to explain how you can get started while respecting your duties as a public servant!
+<!--more-->
+
+In our [previous post]({{site.baseurl}}{% post_url 2019-11-19-working-in-the-open-part-1 %}), we explored what it meant to work in the open and why we should do so by default.
+Adding to our previous post's reasons to work in the open, we should mention that doing so is supported by the [Directive on Open Government](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=28108){:target="_blank"} and even the [Enterprise Architecture Framework](https://wiki.gccollab.ca/GC_Enterprise_Architecture/Framework){:target="_blank"} states that:
```

Comment:
Given that this Blog will be viewed by many outside of ESDC, I suggest that you write "ESDC's Enterprise Architecture Framework" or was it "ESDC's OSS Framework"?

Reply:
Oh, no, that's the GC EARB's own Enterprise Architecture Framework. Essentially,  the overarching framework all other frameworks should aim to align with :smile: